### PR TITLE
add redeem code counts when view each user

### PR DESCRIPTION
**Pull request by lamngue**

### CODE REVIEW

File: gorm/user.go [hunk 1]
```diff
@@ -142,37 +142,54 @@ func GetUsersForAdmin(fn string, gususername string, sortBy string) ([]apiObject
...
 	sql.WriteString(`SELECT
 			u.*,
 			(SELECT COUNT(*)
 			FROM orders o_sub
 			WHERE o_sub.user_id = u.id
-			AND o_sub.status_code >= ?) AS total_orders
+			AND o_sub.status_code >= ?) AS total_orders,
+            (SELECT COUNT(*) 
+            FROM redeemable_codes r
+            WHERE r.redeemed_by_user_id = u.id
```
Comment (duyng404) [inline gorm/user.go]: Please indent these lines equal to the lines above and below

File: gorm/user.go [hunk 2]
```diff
@@ -142,37 +142,54 @@ func GetUsersForAdmin(fn string, gususername string, sortBy string) ([]apiObject
...
+
+//PopulateByIDForAdminDash get info for one user (admin)
+func PopulateByIDForAdminDash(id uint) (apiObjects.AdminUsersStruct, error) {
+	var user apiObjects.AdminUsersStruct
+	err := DB.Raw(`
+		SELECT u.*,
+		(SELECT COUNT(*) 
+            FROM redeemable_codes r
+            WHERE r.redeemed_by_user_id = u.id
+            AND r.status = ?) AS number_of_redeems
```
Comment (duyng404) [inline gorm/user.go]: indent these lines as well

File: gorm/user.go (hunk 1 above)
Comment (lamngue) [reply gorm/user.go]: I did :v

File: gorm/user.go (hunk 2 above)
Comment (lamngue) [reply gorm/user.go]: I did :v

**Pull request merged**